components/account/AccountHeader.vue

```diff
@@ -100,17 +104,18 @@ const isNotifiedOnPost = $computed(() => !!relationship?.notifying)
         </div>
         <div absolute top-18 inset-ie-0 flex gap-2 items-center>
           <AccountMoreButton :account="account" :command="command" />
-          <button
-            v-if="!isSelf && relationship?.following"
-            :aria-pressed="isNotifiedOnPost"
-            :aria-label="t('account.notify_on_post', { username: `@${account.username}` })"
-            rounded-full p2 border-1 transition-colors
-            :class="isNotifiedOnPost ? 'text-primary border-primary hover:bg-red/20 hover:text-red hover:border-red' : 'border-base hover:text-primary'"
-            @click="toggleNotifications"
-          >
-            <span v-if="isNotifiedOnPost" i-ri:bell-fill block text-current />
-            <span v-else i-ri-bell-line block text-current />
-          </button>
+          <CommonTooltip v-if="!isSelf && relationship?.following" :content="getNotificationIconTitle()">
+            <button
+              :aria-pressed="isNotifiedOnPost"
+              :aria-label="getNotificationIconTitle()"
```

Comment:
The reason that I kept the label on the button as just `account.notify_on_post` is because of the way `aria-pressed` works. Screenreaders announce the label and then if the button is "pressed" it will let the user know, which is exactly what we want here. If we change the labels on click then it will no longer do that.

Here's a great article that explains this behavior: https://joshcollinsworth.com/blog/accessible-toggle-buttons#understanding-aria-pressed

Reply:
Thanks for the explanation, makes perfect sense. I can change it back to notify_on_post.

What if the aria-pressed stayed as account.notify_on_post, but the CommonTooltip continued to display the altered text based on current state of the button?

Reply:
The tooltip definitely should display both labels, sorry forgot to mention that. Also just to clarify, I think your naming for the i18n keys is great lets keep them as is.

Reply:
Speaker of naming conventions, to keep all three versions grouped, what do you think about renaming "notify_on_post" as "notifications_on_post_aria"?

Reply:
I was under the impression that the wording for `notifications_on_post_enable` and `notify_on_post` were the exact same. In which case we can get rid of `notify_on_post` entirely.

Reply:
As I pull up the previous code I see that. Ok, leaving it at the two then. Less is more!

Reply:
@rshigg this change was made, however if we decide to add back in the {username} interpolation based on @patak-dev decision, it will change again.

Reply:
We did add the {username} interpolation back in as discussed.